templates/privacy-policy.jade

```diff
@@ -0,0 +1,37 @@
+doctype html
+html(lang='en')
+  head
+    title Privacy Policy | Avalanche Forecasts
+    meta(name='viewport', content='width=device-width, initial-scale=1.0')
+    meta(name='description', content='The easiest way to see avalanche conditions at-a-glance, on your iPhone or Android phone')
+    link(rel='icon', href='/favicon.ico' ,type='image/x-icon') 
+    link(rel='shortcut icon', href='/favicon.ico', type='image/x-icon')
+    link(rel='canonical', href='/')
+    link(href='//netdna.bootstrapcdn.com/twitter-bootstrap/2.1.1/css/bootstrap-combined.min.css', rel='stylesheet')
+    
+    script.
+      (function(d, s, id) {
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = '//connect.facebook.net/en_US/all.js#xfbml=1';
+      fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));
+
+    script.
+      !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");
+  style.
+    .container {
+      margin-top:30px;
+      padding: 0 5%;
+    }
+  body
+    .container
+      h1 Avalanche Forecasts App Privacy Policy
+      p In order to provide our App's services, we must collect some personal information. Protecting your personal information is very important to us. This policy explains what information we collect and how we use it.
+      h2 What information do we collect and how is it used?
+      p When the App requests forecast information from our web servers, our web servers log the request and store the IP address the request was made from. These logs are used for tracking usage as well as maintaining the performance and stability of the web servers. The logs are deleted after 1 year. We share the logs with service providers who also use the information for tracking usage and maintaining the performance and stability of the servers.
```

Comment:
"The logs are deleted after 1 year." -- who is going to do that? might be better to strike this sentence?